docs/outline.toml

```diff
@@ -154,7 +154,7 @@ classes = [
 [pages.engine.result]
 title = "Results"
 module = "prefect.engine.result"
-classes = ["Result", "SafeResult", "NoResultType"]
+classes = ["Result", "SafeResult", "NoResultType", "PrefectResult", "GCSResult", "LocalResult", "S3Result", "AzureResult", "SecretResult", "ConstantResult"]
```

Comment:
The tests are failing because, unfortunately, these all live in `prefect.engine.results` (plural). We can sorry about the API side in a future PR if you don't want to deal with that in this one

Reply:
I believe this is fixed up now!